src/restic/backend/gs/config.go

```diff
@@ -0,0 +1,71 @@
+package gs
+
+import (
+	"path"
+	"strings"
+
+	"restic/errors"
+	"restic/options"
+)
+
+// Config contains all configuration necessary to connect to an gcs compatible
+// server.
+type Config struct {
+	Location    string
```

Comment:
I think Location should be removed from here. According to api,
```
	// Location: The location of the bucket. Object data for objects in the
	// bucket resides in physical storage within this region. Defaults to
	// US. See the developer's guide for the authoritative list.
	Location string `json:"location,omitempty"`
```

So, if user wants to use a different location, they can create they bucket ahead of time. This will be similar to how S3 behaves.